File: API/Src/DTOs/RegisterRequest.cs
namespace ListasAPI.Src.DTOs
{
    public class RegisterRequest
    {
        public string name { get; set; } = string.Empty;
        public string email { get; set; } = string.Empty;
        public string password { get; set; } = string.Empty;
        public string version { get; set; } = string.Empty;
    }
}

File: Aplicativo/app/src/main/java/com/example/listadecompras/models/RegisterRequest.java
package com.example.listadecompras.models;

public class RegisterRequest {
    public String name;
    public String email;
    public String password;
    public String version;

    public RegisterRequest(String name, String email, String password, String version) {
        this.name = name;
        this.email = email;
        this.password = password;
        this.version = version;
    }
}
